samples/custom-bootstrap/gateway-patch.yaml

```diff
@@ -0,0 +1,18 @@
+spec:
```

Comment:
instead of a patch should we give an IOP spec that does the same? all of this should be configurable by IOP right? Or maybe not configmap mount.. might just be secret

Reply:
I am not sure if it works across 1.4, 1.5, 1.6 .
How will iop work in this case?

Reply:
I think IOP override might not work across 1.4, 1.5 and 1.6, specifically 1.4 the API was different

Reply:
1.4 is indeed a little different, it would look like this:
```
gateways:
  components:
    ingressGateway:
      k8s:
        overlays:
```
vs 1.6 version:
```
ingressGateways:
- name: istio-ingressgateway
  k8s:
    overlays:
```

The rest of the syntax for the patching operation (overlays) is the same.

Reply:
Thanks.
We need to make this better for 1.6.next and it makes sense to use it in this way.